### PR TITLE
Two versions of program that shows name and current date.

### PR DESCRIPTION
**Pull request by s0me0nelv**

### CODE REVIEW

File: Lesson 2/EndOfLesson2.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import java.util.*;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+public class EndOfLesson2 {
+    public static void main(String[] args) {
+        EndOfLesson2.showNameAndTime("Eugene", "Bujanov");
```
Comment (algorithm108) [inline Lesson 2/EndOfLesson2.java:6]: Name and Surname are hardcoded rights now, I would like as a User enter name and surname by myself

**Review by algorithm108 — CHANGES_REQUESTED** [review]
Please change behavior of programme that user can enter name and surname manually

File: Lesson 2/EndOfLesson2.java (hunk 1 above)
Comment (s0me0nelv) [reply Lesson 2/EndOfLesson2.java:6]: added EndOfLesson2ver3.java where it is possible to pass Name and Surname from the command line

**Review by algorithm108 — APPROVED** [review]

File: Lesson 2/EndOfLesson2.java (hunk 1 above)
Comment (Aadumans) [reply Lesson 2/EndOfLesson2.java:6]: I would suggest you need to google up method or methods how to read input from console...

Comment (s0me0nelv) [reply Lesson 2/EndOfLesson2.java:6]: In "Files changed" tab you can find EndOfLesson2ver2.java where program asks to input name and surname using Scanner and also EndOfLesson2ver3.java where we can by running program in console pass argument with our Name and Surname to console. Example to run in console: java EndOfLesson2ver3 "Eugene Buyanov".

**Pull request merged**